tests/sqlx/lexer.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { constructSyntaxTree, parseSqlx, lexer } from "@dataform/sqlx/lexer";
+import { expect } from "chai";
+
+describe("@dataform/sqlx", () => {
+
+  describe("lexer", () => {
+    it("sqlx_single_backslash_stays_as_double", () => {
```

Comment:
nit: this should really read as a phrase, "replaces single backslashes with double backslashes"